<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleStatus</class>
 <widget class="QWidget" name="ModuleStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>114</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>80</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="moduleLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkHistogramBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Histogram</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkActiveBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Active</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>54</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="moduleType">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>MCPD-8+
 </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="moduleVersion">
     <property name="text">
      <string>1.0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="moduleStatus">
     <property name="text">
      <string>online</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkHistogramBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>ModuleStatus</receiver>
   <slot>histogramSlot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkActiveBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>ModuleStatus</receiver>
   <slot>activeSlot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>clicked(quint8)</signal>
  <signal>histogram(quint8, bool)</signal>
  <slot>histogramSlot(bool)</slot>
  <slot>activeSlot(bool)</slot>
 </slots>
</ui>
